### PR TITLE
Remove extra eslintrc and fix style

### PR DESCRIPTION
**Pull request by kewisch**
There was an extra eslintrc in the calendar directory which was blocking some rules. I didn't pull all the rules over, strangely there are some pretty basic rules that don't seem to be enabled in the recommended set, but I need to stop somewhere :)

### CODE REVIEW

File: NotificationBox/background.js [hunk 1]
```diff
@@ -6,9 +6,8 @@ messenger.NotificationBox.onButtonClicked.addListener((windowId, notificationId,
   if (["btn-keep"].includes(buttonId)) {
     console.log("Box will not close, as long as one listener returns {close:false}.");
     return { close: false };
-  } 
+  }
     return { close: true };
```
Comment (jobisoft) [inline NotificationBox/background.js:10]: Strange that the indent here is not fixed by the linter. It is a left-over from the `else` being removed. Any idea?

File: calendar/experiments/calendar/parent/ext-calendar-provider.js [hunk 2]
```diff
@@ -598,12 +596,11 @@ this.calendar_provider = class extends ExtensionAPI {
                 );
                 if (props?.error) {
                   return { error: props.error };
-                } else {
+                }
                   if (props?.type) {
```
Comment (jobisoft) [inline calendar/experiments/calendar/parent/ext-calendar-provider.js:600]: Strange indent here as well.

**Comment by kewisch** [comment]
So indent was also not enabled by mozilla/recommended. I do wonder what other fairly obvious rules might not be enabled then. I don't really want to go back to the mega eslintrc. Do you want to do this piece by piece, or do an audit of all rules?

**Comment by jobisoft** [comment]
I would like to review this as it is. If we find issues later, we can always file a new PR.

**Review by jobisoft — APPROVED** [review]
This looks good.

**Pull request merged**